src/mozillavpn.cpp

```diff
@@ -698,6 +709,14 @@ void MozillaVPN::createSupportTicket(const QString& email,
   });
 }
 
+void MozillaVPN::launchPlayStore() {
```

Comment:
Wondering if this method should be included in a `#ifdef MVPN_ANDROID`. Do we need to expose it elsewhere?

Reply:
I wrapped in #ifdef.